style/sass/README.md

```diff
@@ -32,10 +32,9 @@
 * Use alphabetical order for declarations.
 * Place `@extends` and `@includes` at the top of your declaration list.
 * Place media queries directly after the declaration list.
-* Place concatenated selectors second.
-* Place pseudo-states and pseudo-elements third.
-* Place nested elements fourth.
-* Place nested classes fifth.
+* Place pseudo-states and pseudo-elements second.
```

Comment:
I know this PR focuses on declaration ordering, but I noticed that in [this PR](https://github.com/thoughtbot/guides/pull/519/files), you corrected `pseudo-states` to `pseudo-classes`. Is that a change that should happen for this section, as well?

Reply:
The two PRs are editing the same content, so once we merge we merge into `master` it will all be up-to-date with that change to `pseudo-classes`.